lib/pretty-print.js

```diff
@@ -1,5 +1,118 @@
+/* eslint-disable no-multi-spaces */
```

Comment:
Can you please add `'use strict'` at the top?

Reply:
Done!